doc/tutorials/using_windows_as_uos.rst

```diff
@@ -0,0 +1,375 @@
+   "NUC7i5DNHE", "DNKBLi5v.86A.0060.2018.1220.1536", "`link <https://downloadcenter.intel.com/download/28885?v=t>`__"
+
+ACRN Service OS Setup
+*********************
+You may refer to the steps in :ref:`getting-started-apl-nuc` for
+Intel NUC to setup ACRN on KBL NUC. After following the steps in that guide,
+you should be able to launch a Clear Linux UOS successfully.
+
+Setup for Using Windows as Guest OS
+***********************************
+All the patches to support WaaG have been upstreamed, you can download them
+from the acrn-hypervisor repository.
+
+Build ACRN EFI Images
+=====================
+   #. Follow the steps described at :ref:`getting-started-building` to set up the build environment
+   #. Use ``make`` command to compile the ``acrn.efi`` and ``acrn-dm``::
+
+      $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+      $ cd acrn-hypervisor
+      $ make FIRMWARE=uefi BOARD=kbl-nuc-i7
+
+   #. You will get the outputs from::
+
+      $ build/hypervisor/acrn.efi
+      $ build/devicemodel/acrn-dm
+
+   #. Replace the ``acrn.efi`` and ``acrn-dm`` on your NUC:
+
+      a. Login ACRN Service OS and then ``mount`` EFI partition to ``/boot``
+      #. ``scp`` the ``acrn.efi`` and ``acrn-dm`` from your host::
+
+         # scp <acrn.efi from your host> /boot/EFI/acrn/
+         # scp <acrn-dm from your host> /usr/bin/
+         # chmod +x /usr/bin/acrn-dm && sync
+
+Build Service OS Kernel
+=======================
+   #. Follow the steps described at :ref:`getting-started-building` to set up the build environment
+   #. Follow the steps below to build ACRN kernel::
+
+      $ WORKDIR=`pwd`;
+      $ JOBS=`nproc`
+      $ git clone -b master https://github.com/projectacrn/acrn-kernel.git
+      $ cd acrn-kernel && mkdir -p ${WORKDIR}/{build,build-rootfs}
+      $ cp kernel_config_uefi_sos ${WORKDIR}/build/.config
+      $ make olddefconfig O=${WORKDIR}/build && make -j${JOBS} O=${WORKDIR}/build
+      $ make modules_install INSTALL_MOD_PATH=${WORKDIR}/build-rootfs O=${WORKDIR}/build -j${JOBS}
+
+Update Kernel on KBL NUC
+========================
+   #. Copy new kernel image (bzImage) and it's modules to target machine::
+
+      # scp <your host>:$WORKDIR/build/arch/x86/boot/bzImage /boot/bzImage
+      # scp -r <your host>:$WORKDIR/build-rootfs/lib/modules/* /lib/modules/
+      # cp /boot/loader/entries/acrn.conf  /boot/loader/entries/acrngt.conf
+
+   #. Modify ``acrngt.conf`` to the content as given below:
+
+      .. code-block:: none
+
+          title The ACRNGT Service OS
+          linux /bzImage
+          options console=tty0 console=ttyS0 root=/dev/sda3 rw rootwait ignore_loglevel no_timer_check consoleblank=0 i915.nuclear_pageflip=1 i915.avail_planes_per_pipe=0x010101 i915.domain_plane_owners=0x011100001111 i915.enable_gvt=1 i915.enable_conformance_check=0 i915.enable_guc=0 hvlog=2M@0x1FE00000
+
+      .. note:: You should change ``/dev/sda3`` to your file system partition.
```

Comment:
Please left-align notes:
.. note:: You should change ``/dev/sda3`` to your file system partition.